pkg/operator/latencyprofilecontroller/helpers/helpers.go

```diff
@@ -0,0 +1,115 @@
+package helpers
```

Comment:
Put these revision matcher files inside `pkg/operator/latencyprofilecontroller` no need to multiple packages. Call it `revisionconfigmatcher.go` explicitly.